### ISSUE TITLE
CreatePasswordResetsTable

### ISSUE THREAD

**Issue by baqirfarooq** [darryldecode/laravelbackend#2]
CreatePasswordResetsTable error on migrate

**Comment by darryldecode** [comment]
Hello @baqirmemon ,

Did you follow exactly on the docs for installation guide?

http://laravelbackend.com/docs/1.0/?page=installation.php

**Comment by bsesic** [comment]
I fixed the issue with dropping the table from previous migrations.

**Comment by darryldecode** [comment]
Yes, it is recommended a fresh installation with no tables migrated yet as it will conflict with Laravel's bundled default migrations.. It is even stated in the installation docs in STEP 3 to delete first all Laravel's default migration files.

**Comment by bsesic** [comment]
Add `Schema::dropIfExists('password_resets');` directly after `public function up(){` in the migration.

**Comment by darryldecode** [comment]
30b988b8c9089f3455e145c4d626a26aa0a8d005

**Comment by reypm** [comment]
@darryldecode I am running the same issue. I have checked DB and there is only one `migrations` table. I have checked `/databases/migrations` directory and I can see two files in there:
- `2014_10_12_000000_create_users_table.php` 
- `2014_10_12_100000_create_password_resets_table.php`

Should I delete those two before run the command?

```
php artisan vendor:publish --provider="Darryldecode\Backend\BackendServiceProvider"
```

I can see a patch here, it's merged? I installed a fresh copy and I can't see this line `Schema::dropIfExists('password_resets');` on migrations files as it's in patch. What I am missing here?
